gc.go

```diff
@@ -52,6 +52,10 @@ type GraphicContext interface {
 	GetFontData() FontData
 	// GetFontName gets the current FontData as a string
 	GetFontName() string
+	// SetGlyphCache changes the glyph cache backend used by the GraphicContext
+	SetGlyphCache(cache GlyphCache)
```

Comment:
I think this function don't have to be in the Gc interface. It is useful only in some implementation of Gc not all.